tools/chrome-debug-extension/src/telemetryViewerPopup.tsx

```diff
@@ -113,7 +113,7 @@ export const TelemetryViewerPopup = (): React.ReactElement => {
     React.useEffect(() => {
         versionCheck();
 
-        let configurationTypeToSet = undefined;
+        let configurationTypeToSet: ConfigurationType = undefined;
```

Comment:
TS 3.x actually complained about this and set the type of this variable to string | undefined -- which was wrong.
While this doesn't affect anyone consumers I fixed anyway so that i could validate that the entire build works.